docs/plugin/cue.md

```diff
@@ -87,9 +87,7 @@ A variable migration file looks like the following:
 if #var.type == "custom" || #var.type == "interval" {
     kind: "StaticListVariable"
     spec: {
-        values: [ for _, option in #var.options if option.value !~ "^\\$__.*$" { // the if filters out the grafana global vars that'd be causing validation issues later (e.g "__auto_interval_sampling is used but not defined")
-            option.value
-        }]
+        values: strings.Split(#var.query, ",")
```

Comment:
Here this is just an example to explain the plugin format, so we don't care if that's not the real logic used by the StaticListVariable plugin in reality (better to keep it simple in this part of the doc).